src/modules/staking/services/staking.service.ts

```diff
@@ -106,6 +115,37 @@ export class StakingService {
         });
     }
 
+    async getAllAccumulatedRewards(
```

Comment:
I would move these methods to abi service. It should be at the same level with the single address methods

Reply:
done